### ISSUE TITLE
GitCommitBear: Check for WIP: shortlog

### ISSUE THREAD

**Issue by virresh** [coala/coala-bears#2471]
Currently the bear only yields a result in case the shortlog starts with
"[wip] "
It should also check for "WIP: "

**Comment by rohanrk** [comment]
May I be assigned to this issue?

**Comment by jayvdb** [comment]
@rohanrk , @virresh , it looks like the code was already checking for `WIP:` , but in a slighly obscure way.

And it wasnt properly supported, as there was another related bug with relatation to imperative tense on `WIP: ` patches , and the PR fixes that other bug.

**Comment by rohanrk** [comment]
When I tested the bear, it failed WIP checks when I had ```WIP:```

**Comment by jayvdb** [comment]
I think it failed the imperative checks, not the wip checks.
If so, it was possible to configure GitCommitBear to use `WIP: ` , but only if the  imperative checks were disabled.

**Comment by rohanrk** [comment]
Yeah. this is because the imperative check modified the shortlog and the WIP check occurred after the imperative check.
